### ISSUE TITLE
What‘s the difference between CVPR submission template and ICCV submission template?

### ISSUE THREAD

**Issue by linhuixiao** [apoorvkh/cvpr-latex-template#4]
What‘s the difference between CVPR submission template and ICCV submission template?  Is there a difference besides the name of the meeting?

**Comment by apoorvkh** [comment]
Hi @linhuixiao. From what I understand, CVPR and ICCV are supposed to use the same template. For CVPR 2022, this template was updated ("modernized") after a long time (15 years): [MCG-NKU/CVPR_Template](https://github.com/MCG-NKU/CVPR_Template). CVPR 2022 and 2023 followed this template, but ICCV 2023 (accidentally?) reverted to the old template. I am not sure if there are significant cosmetic changes, but I am investigating.

**Comment by apoorvkh** [comment]
I contacted Prof. Ming-Ming Cheng (who authored [the most recent template update](https://github.com/MCG-NKU/CVPR_Template)) and he emphasized that both versions "should result in the same PDF".

So, according to him, it should be okay to use this repo for submissions to ICCV 2023.

I will close this issue later, once I verify their differences at a granular level.

**Comment by linhuixiao** [comment]
Thanks for your efforts, this is a interesting thing, and it may save some typesetting effort for many researchers' conference submissions.

**Comment by apoorvkh** [comment]
I compared the style files between the "modernized" CVPR template and (older) ICCV 2023 template. I can't find any substantial cosmetic differences! Closing this issue.

tl;dr: the template in this repo should indeed be compatible with submissions to ICCV 2023.
